examples/detect_onsets_presets.py

```diff
@@ -11,32 +11,39 @@
 
 import sys
 import argparse
+import os
 import matplotlib.pyplot as plt
 from carat import annotations, audio, display, onsets, util
 
 
-def onset_detection(input_file, output_file, json_file, preset_name):
+def onset_detection(input_file, output_file, preset_name,
+                    json_file, verbose):
     '''Onset detection
 
     :parameters:
       - input_file : str
           path to input audio file (wav, mp3, m4a, flac, etc.)
       - output_file : str
           path to the output csv file
-      - json_file : str
-           path to the json file of presets
       - preset_name : str
             name of the preset to load (e.g. "chico")
+      - json_file : str
+           path to the json file of presets. Default: carat/presets/onsets.json
+      - verbose : bool
+           if True print values of parameters in chosen preset. Default: True
```

Comment:
The default value is True because of the command line arguments parser. You could add "verbose=True" in the definitions of the onset_detection function to make it consistent with the doc string.

Reply:
"with the doc string"?
Yep, I thought about setting the default in the function's definition, but when I saw the default field in the parser chose to use that. I think it should be one or the other, not both or mixed, so we just have to decide which we prefer.